src/plugins/remoteClock/RemoteClock.js

```diff
@@ -77,7 +77,7 @@ export default class RemoteClock extends DefaultClock {
         this._subscribe();
       })
       .catch((error) => {
-        throw new Error(error);
+        throw error;
```

Comment:
shouldn't we just throw the error here instead of catching at all?

Reply:
lol, you're right, thank you. The original catch block was wrapping the error for some reason and masking the original traceback, so I unwrapped it but in the process rendered it completely redundant :)